File: NodeJS/Native.js
var http = require('https');

var token = '56a351796b80339c09ff1b76';
var requestBody = {
  'collector': token,
  'from': {
    'name': 'Ezequias Dinella',
    'email': 'ezequias@binds.co'
  }
};

var options = {
  'method': 'POST',
  'hostname': 'app.binds.co',
  'port': null,
  'path': '/api/seeds',
  'headers': {
    'content-type': 'application/json; charset=UTF-8'
  }
};

var req = http.request(options, function(res) {
  var chunks = [];
  res.on('data', function(chunk) {
    chunks.push(chunk);
  });
  res.on('end', function() {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write(JSON.stringify(requestBody));
req.end();

File: Go/Native.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "https://app.binds.co/api/seeds"

	payload := strings.NewReader("{\"collector\":\"56a351796b80339c09ff1b76\",\"from\":{\"name\":\"Ezequias Dinella\",\"email\":\"ezequias@binds.co\"}}")

	req, _ := http.NewRequest("POST", url, payload)

	req.Header.Add("content-type", "application/json; charset=UTF-8")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}
